microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -195,35 +235,46 @@ JavaScript debugging is now built in to Visual Studio Code. Starting with Versio
             }
         ]
     }
-    
-## Console integration
-If you launch the extension from the Run and Debug workflow, the [Debug Console of Visual Studio Code][DebugConsoleVSCode] gives you all the functions of the [Microsoft Edge Developer Tools Console][EdgeDevToolsConsole]. You have direct access to the Window object of the instance of Edge and can use all the [Console Utilities][ConsoleUtilities].
+    ```
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
-   Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow  
-:::image-end:::
+1.  Change `http://localhost:8080` in the above code and make sure the variable `{workspaceFolder}` resolves.
 
-## Getting in touch with the Microsoft Edge DevTools for Visual Studio Code extension team  
+1.  Select the **Inspect** icon.  If you haven't installed the Microsoft Edge DevTools extension for Visual Studio Code, the **Extensions** tab opens and automatically displays the extension to install. <!-- this step was stray; move? -->
 
-Send your feedback by [filing an issue][GithubMicrosoftVscodeEdgeDevtoolsNewIssue] on the [microsoft/vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.  
+#### See also
 
-If you want to help make <!--the Microsoft Edge DevTools for Visual Studio Code -->this extension better, your contributions are welcome.  Find everything you need to get started in the [microsoft/vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension. 
+*  [Launch configurations][launchJSON] in the _Debugging_ article for Visual Studio Code.
 
-<!--links -->  
 
-[VisualstudioCode]: https://code.visualstudio.com "Visual Studio Code"  
-[VisualStudioCodeDocs]: https://code.visualstudio.com/Docs "Documentation | Visual Studio Code"   
+<!-- ====================================================================== -->
+## Console integration
+
+If you launch the extension from the Run and Debug workflow, the [Debug Console of Visual Studio Code][DebugConsoleVSCode] gives you all the functions of the DevTools [Console][EdgeDevToolsConsole].  You have direct access to the Window object of the instance of Edge and can use all the [Console Utilities][ConsoleUtilities].
+
+:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="The DevTools Console is available when the extension is launched from a Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
```

Comment:
This image is too small.  Re-create or find a bigger original.

Reply:
@codepo8 - Do you have bigger versions of the .png files for this article?  They are somewhat too small, and clicking the images ("lightbox" view) doesn't make them bigger.

Reply:
Notified writer to add article to list to update screenshots.